apps/web/src/views/Swap/MMLinkPools/components/MMCommitButtonV2.tsx

```diff
@@ -63,6 +61,7 @@ export function MMSwapCommitButtonV2({
   currencyBalances,
   recipient,
   onUserInput,
+  setLock,
```

Comment:
I'm going to revisit my previous comment here https://github.com/pancakeswap/pancake-frontend/pull/9274/#discussion_r1520802664

It's not quite readable to me to have a `setLock` prop on a button. Can we redesign the interface?